.github/workflows/suite-sync.yml

```diff
@@ -0,0 +1,47 @@
+name: Update GitBook revision in Suite
+
+on:
+  push:
+
+
+jobs:
+  update-revision-in-suite:
+    if: github.event.head_commit.committer.username == 'gitbook-bot'
+    
+    env:
+      SUITE_REPO: "trezor/trezor-suite"
+      SUITE_BRANCH: "develop"
+      FILEPATH: "./packages/suite-data/src/guide/constants.ts"
+      PR_BRANCH: "chore/update-suite-guide"
```

Comment:
i would instead used different branch name for every pr so something like `chore/update-suite-guide-${{ github.sha }}`

Reply:
Static branch name has the advantage of a stable dev URL which can be bookmarked by the writers. So, unless it causes any issues, I prefer the current variant.